cmd/juju/action/list.go

```diff
@@ -35,17 +35,17 @@ type listCommand struct {
 }
 
 const listDoc = `
-List the actions available to run on the target application, with a short
-description.  To show the full schema for the actions, use --schema.
+List the functions available to run on the target application, with a short
+description.  To show the full schema for the functions, use --schema.
 
 Examples:
-    juju list-actions postgresql
-    juju list-actions postgresql --format yaml
-    juju list-actions postgresql --schema
+    juju list-functions postgresql
```

Comment:
Why is the primary command "functions" and "list-functions" is the alias, yet we show the list command here?

Reply:
was existing code, fixed